### ISSUE TITLE
[EN][Quest] More Dense Grinding Stones

### ISSUE THREAD

**Issue by MichaelArchangelo** [Atlantiss/BugTracker#7758]
The quest More Dense Grinding Stones (http://www.wowhead.com/quest=7939) doesn't give reputation for Darkmoon Faire.

**Comment by Aival94** [comment]
This quest give reputation till you reach 2000/6000 of Friendlty reputation. If you have reached this rep it is working correct.

**Comment by MichaelArchangelo** [comment]
No, it doesn't give reputation at all. I know we're technically on 4.3.4, but Darkmoon Island isn't here yet.

**Comment by user1011** [comment]
How much reputation should the quest give?

**Comment by user1011** [comment]
Requires more testing on Atlantiss core, since Trinity core has Darkmoon Island.

**Comment by Aival94** [comment]
This quest used to give 250 rep till you reach 2000/6000 friendly reputation.
